onnx_chainer/export.py

```diff
@@ -43,8 +43,6 @@ def convert_parameter(parameter):
             'The type of parameter is unknown. It should be either Parameter '
             'or Variable or ndarray, but the type was {}.'.format(
                 type(parameter)))
-    if array.shape == ():
```

Comment:
Nice! I haven't realized this and I was wondering why `param_shape = (1,)` thing is necessary.